larpandoracontent/LArHelpers/LArMCParticleHelper.cc

```diff
@@ -131,6 +139,20 @@ void LArMCParticleHelper::GetTrueNeutrinos(const MCParticleList *const pMCPartic
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+void LArMCParticleHelper::GetTrueTestBeamParticles(const MCParticleList *const pMCParticleList, MCParticleVector &trueTestBeamParticles)
+{
+    for (const MCParticle *const pMCParticle : *pMCParticleList)
+    {
```

Comment:
This could call `LArMCParticleHelper::IsTriggeredBeamParticle` so that the definition, with its use of a specific nuance value, etc., is implemented in only one function.